tests/test_external_search.py

```diff
@@ -943,7 +1027,8 @@ def assert_order(sort_field, order, **filter_kwargs):
             """
             expect = self._expect_results
             facets = Facets(
-                self._default_library, None, None, order=sort_field, order_ascending=True
+                self._default_library, Facets.COLLECTION_FULL,
+                Facets.AVAILABLE_ALL, order=sort_field, order_ascending=True
```

Comment:
The `Facets` object can now impose additional filters on test results, so when we create one to test sort order we need to set the facets to values that won't actually filter anything out.